WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -93,12 +118,15 @@ class CommentContentTableViewCell: UITableViewCell, NibReusable {
 
         updateLikeButton(liked: comment.isLiked, numberOfLikes: comment.numberOfLikes())
 
-        // configure comment content
+        // Configure feature availability.
+        isReactionEnabled = !comment.isReadOnly()
+        isCommentLikesEnabled = comment.blog?.supports(.commentLikes) ?? false
```

Comment:
The old comment detail shows Like based on what the blog and the comment support:
`cell.isLikeEnabled = [self.comment.blog supports:BlogFeatureCommentLikes] && self.userCanLikeAndReply;`

I assume the new one should be the same, that is:
`isCommentLikesEnabled = comment.blog?.supports(.commentLikes) ?? !comment.isReadOnly()`

Reply:
Ah yep. Actually, the comment button is contained in a view that has its visibility controlled through `isReactionEnabled`. So, when the value of `!comment.isReadOnly()` is `false`, both the like and comment buttons will be hidden regardless of whether they're visible or not.

But I guess this behavior is not very clear just from a single glance unless one also checks out the XIB – and things may change in the future and it might introduce a bug. I'll adjust this soon! Thanks. 🙂

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17147/commits/336c31fe0f248044a21061303ad718ee66ca0e4c!